src/org/openj9/envInfo/JavaInfo.java

```diff
@@ -229,7 +229,12 @@ public void checkCRIU() {
         String isCRIUCapable = System.getProperty("org.eclipse.openj9.criu.isCRIUCapable");
         if ((isCRIUCapable != null) && isCRIUCapable.equals("true")) {
             detectedTfs.add("CRIU");
-            detectedTfs.add("CRAC");
+            String osArch = System.getProperty("os.arch");
+            if ("amd64".equalsIgnoreCase(osArch)) {
+                // CRIU is only supported on Linux.
```

Comment:
Do you mean to have the comment about CRIU within this block or above L231 ?

Reply:
Sure, moved the CRIU comment up, and also did minor refactoring.